islandora_checksum.module

```diff
@@ -100,12 +100,15 @@ WHERE {
     ?object <fedora-model:createdDate> ?date .
   } UNION {
     ?object <{$islandora_rels_ext}isPageOf> ?book .
+    ?object <fedora-model:createdDate> ?date .
     ?book <fedora-rels-ext:isMemberOfCollection> <info:fedora/$pid> .
   } UNION {
     ?object <fedora-rels-ext:isConstituentOf> ?compound .
+    ?object <fedora-model:createdDate> ?date .
     ?compound <fedora-rels-ext:isMemberOfCollection> <info:fedora/$pid> .
   } UNION {
     ?object <fedora-rels-ext:isMemberOf> ?issue .
+    ?object <fedora-model:createdDate> ?date .
```

Comment:
Not terribly concerned, but I think this could by DRY'd up and pushed outside of the union, something like:
```
PREFIX xs: <xml-schema:>
SELECT DISTINCT ?object ?date
FROM <#ri>
WHERE {
   {
     ?object <fedora-rels-ext:isMemberOfCollection> <info:fedora/$pid> .
   } UNION {
     ?object <{$islandora_rels_ext}isPageOf> ?book .
     ?book <fedora-rels-ext:isMemberOfCollection> <info:fedora/$pid> .
   } UNION {
     ?object <fedora-rels-ext:isConstituentOf> ?compound .
     ?compound <fedora-rels-ext:isMemberOfCollection> <info:fedora/$pid> .
   } UNION {
     ?object <fedora-rels-ext:isMemberOf> ?issue .
     ?issue <fedora-rels-ext:isMemberOf> ?newspaper .
     ?newspaper <fedora-rels-ext:isMemberOfCollection> <info:fedora/$pid> .
   }
  ?object <fedora-model:createdDate> ?date .
  [... whatever ending ...]
```